packages/forms/frontend/sirius-components-forms/src/extensions/PropertySectionLabelDecoratorExtensionPoint.tsx

```diff
@@ -0,0 +1,27 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo and others.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+import { ComponentExtensionPoint } from '@eclipse-sirius/sirius-components-core';
+import { GQLWidget } from '../form/FormEventFragments.types';
+
+export type PropertySectionLabelDecoratorProps = {
+  editingContextId: string;
+  formId: string;
+  widget: GQLWidget;
+};
+
+export const propertySectionLabelDecoratorContributionsExtensionPoint: ComponentExtensionPoint<PropertySectionLabelDecoratorProps> =
+  {
+    identifier: 'widget#propertySectionLabelDecorator',
```

Comment:
Please place the extension point in a file named `NameOfTheComponentExtensionPoints.tsx` and use the name of the component in the identifier. Here something like `propertySectionLabel#decorator` is more appropriate than `widget#propertySectionLabelDecorator`